eapp/public/js/acquisitionform.js

```diff
@@ -237,51 +272,65 @@ function add_term_to_form(termform){
     <input class="form-control" type="text" placeholder="ExperimentID" id="ndar-'+selectedFields.length+'">\
   </div>\
   </div>')
+
+//end of addTerms function
 }
 
 function saveAqInfo(e){
-  e.preventDefault()
-  saveObj['objID'] = ''
-  for (let i=0; i<=selectedFields.length; i++){
-    //let lb =$('label[for="ndar-' + i + '"]').html()
-    let lb=$('label[for="ndar-' + i + '"]').attr('title')
-    console.log('lb1:', lb)
-    saveObj[lb] = $("#ndar-"+ i).val() || $("input[type='radio'][name='option"+i+"']:checked").val()
-    console.log('saveObj[lb]:',saveObj[lb])
+  if(fieldsCorrect == false){
+    e.preventDefault()
```

Comment:
e.preventDefault() could be moved from inside if/else block to before the block.